SPEC/FILES.md

```diff
@@ -539,6 +539,252 @@ pull(
 
 A great source of [examples][] can be found in the tests for this API.
 
+#### `cp`
```

Comment:
The MFS specific commands need a separator and an introduction to what MFS is.

Reply:
Okay @diasdavid. It makes sense. Although, there is a command (`ls`) who was already spec'ed and it says that it is only for MFS **but** that it should be implemented globally. [See](https://github.com/ipfs/interface-ipfs-core/blob/a62686ff8834dc3f7ac7bc4fde1c302bdf8dfc01/SPEC/FILES.md#javascript---ipfslsipfspath-callback).